pom.xml

```diff
@@ -437,6 +525,7 @@
         <carbon.identity.local.auth.api.version>2.5.10</carbon.identity.local.auth.api.version>
         <identity.inbound.auth.oauth.version.range>[7.0.26, 7.0.62)</identity.inbound.auth.oauth.version.range>
         <org.wso2.orbit.nimbus.version>7.3.0.wso2v1</org.wso2.orbit.nimbus.version>
+        <oltu.version>1.0.0.wso2v3</oltu.version>
```

Comment:
this is already defined in L519 right?

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/8/commits/55101612f0aa99105f83a6d7d3d648000eb13112